book/website/afterword/glossary.md

```diff
@@ -170,14 +177,23 @@ Digital Object Identifier
 
 ```{glossary}
 
+End to End Test
+  A test that runs the program from beginning to end and verifies that the output is correct.
+
 Epistemology
-  Theory of knowledge and deals with how knowledge is gathered and from which sources. In research terms your view of the world and of knowledge strongly influences your interpretation of data and therefore your philosophical standpoint should be made clear from the beginning. (Source: [Post by Nicole Brown](https://www.nicole-brown.co.uk/epistemology/))
+  Theory of knowledge and deals with how knowledge is gathered and from which sources.
+  In research terms your view of the world and of knowledge strongly influences your interpretation of data and therefore your philosophical standpoint should be made clear from the beginning.
+  (Source: [Post by Nicole Brown](https://www.nicole-brown.co.uk/epistemology/))
 
 Equitable, Diverse and Inclusive Practices
   Ensuring scholarship is open to anyone without barriers based on factors such as race, background, gender, and sexual orientation.
 
-End to End Test
-  A test that runs the program from beginning to end and verifies that the output is correct.
+Ethical Source Software
+  These licenses were created as 'traditional' {term}`open source <Open Source Software>` license make no restictions on the uses to which  the software can be put including unethical uses.
```

Comment:
This might soften the phrasing a little from the first version https://github.com/the-turing-way/the-turing-way/pull/3889/commits/8cc47070b05e2c4b9a1da4cd87afb7fa1a002578